packages/loader/driver-utils/src/test/parallelRequests.spec.ts

```diff
@@ -47,7 +47,7 @@ describe("Parallel Requests", () => {
 						length = Math.min(length, payloadSize / 2 + 1);
 						break;
 					case HowMany.TooMany:
-						length += 2;
+						length = 2 * length + 2;
```

Comment:
maybe go further here, like 2.5 * length, to make it more interesting :)

Reply:
I checked, it works. So leaving as is since 2.* length + 2 is enough to run into while loop.